oval-schemas/oval-definitions-schema.xsd

```diff
@@ -938,6 +939,16 @@
             <xsd:group ref="oval-def:ComponentGroup"/>
         </xsd:sequence>
     </xsd:complexType>
+    <xsd:complexType name="MergeFunctionType">
+        <xsd:annotation>
+            <xsd:documentation>The merge function takes one or more components and merges them together into a single string, with an optional delimiter.  For example if data from one registry multi-size value contains values of "abc" and "def".  The merge function will resolve to a local_variable with the value of 'abcdef'.  If an optional delimiter of ',' was used, the merge function would resolve to a local_variable with the value of 'abc,'def'</xsd:documentation>
```

Comment:
Edited slightly:

The merge function takes one or more components and merges them together into a single string value, optionally including a delimiter string. For example, if data from one registry multi-size value contains values of "abc" and "def", the merge function operated on those values would resolve to a string with the value of 'abcdef'. If an optional delimiter of ',' was used, the merge function would resolve to a value of 'abc,'def'.